### ISSUE TITLE
We can no longer use any version of v3.x because of incompatible upgrade/downgrade requirements

### ISSUE THREAD

**Issue by jonseymour** [hashicorp/terraform-provider-aws#27460]
### Description

I am still using v3.x:

I was forced to go to v3.75.2 by this issue - #24874, but now this issue is forcing me to go to v3.73.0 - #22866 - obviously I can't do both.

It seems the only supported solution is to go to v4.x although in practice back-porting #24874 onto v3.73.0 or reverting the breaking change to v3.74.0  would be pragmatic from my point of view.

Is there any chance at all a new v3.73.x or new v3.75.x version could be cut which avoids the issue introduced in v3.74.x?

### References

#24874 - forces upgrade to v3.75.2
#22866 - forces downgrade to v3.73.0

### Would you like to implement a fix?

Yes

**Comment by jonseymour** [comment]
I am happy to backport and make available the lambda/nodejs16.x fix from v3.75.x onto v3.73.x to produce a new version of v3.73.x if there is interest

**Comment by ewbankkit** [comment]
@jonseymour Thanks for raising this issue 👏.

We are planning on a **v3.x.x** release in the next couple of weeks in support of an upcoming AWS service change.
Would a backport of #22866 onto `release/3.x` (and hence into the upcoming release) work for you?

**Comment by jonseymour** [comment]
Yes, that works for me, thank you! Feel free to close this issue.
